tests/function.py

```diff
@@ -3850,3 +3846,12 @@ def test_send_fds_from_socket(self):
 
   def test_recv_fds_from_socket(self):
     self.assertOnlyIn((3, 9), detect("from socket import recv_fds"))
+
+  def test_create_module_from_importlib_machinery_FrozenImporter(self):
+    self.assertOnlyIn((3, 4), detect("import importlib.machinery.FrozenImporter\nimportlib.machinery.FrozenImporter.create_module()"))
+
+  def test_exec_module_from_importlib_machinery_FrozenImporter(self):
+    self.assertOnlyIn((3, 4), detect("import importlib.machinery.FrozenImporter\nimportlib.machinery.FrozenImporter.exec_module()"))
```

Comment:
This line is too long. Please run `make check-all` and you'll probably find other cases.

For some reason, my TravisCI checks didn't execute for your PR..